src/Symfony/Component/Stopwatch/Stopwatch.php

```diff
@@ -103,7 +103,7 @@ public function stopSection($id)
      *
      * @return StopwatchEvent
      */
-    public function start($name, $category = null)
+    public function start(string $name, string $category = null)
```

Comment:
`@param` annotations can be removed as they're redundant now.

Reply:
The same is true for many other phpdocs that can be adjusted now.